activesupport/lib/active_support/encrypted_file.rb

```diff
@@ -48,7 +48,8 @@ def read
 
     def write(contents)
       IO.binwrite "#{content_path}.tmp", encrypt(contents)
-      FileUtils.mv "#{content_path}.tmp", content_path
+      path = content_path.symlink? ? content_path.realpath : content_path
+      FileUtils.mv "#{content_path}.tmp", path
```

Comment:
Why do we put this check here versus when we assign content_path initially?

Reply:
Thank you for the review. I'll fix it.